server/starters/web-starter/src/main/java/com/oceanbase/odc/service/iam/auth/WebAuthenticationFacade.java

```diff
@@ -24,5 +25,6 @@
  */
 @Profile("alipay")
 @Service
+@ConditionalOnMissingBean(AuthenticationFacade.class)
```

Comment:
we may use some properties to control this bean's init

Reply:
okay, i'll  make this change